src/doodle/doodleController.js

```diff
@@ -90,11 +91,12 @@ export class DoodleController {
     return this._doodleable;
   }
 
-  saveLines() {
-    this._savedLines = [...this._newLines, ...this._savedLines];
-    this._newLines = [];
-    this.render();
-  }
+  //TODO: can we get rid of this?
```

Comment:
Considering it is no longer in use I think we should just kill it. We can always bring it back later if we end up needing it

Reply:
ahh good catch